src/components/Modal/Modal.test.tsx

```diff
@@ -149,4 +149,12 @@ describe('Modal Component', () => {
 
     expect(baseElement).toMatchSnapshot()
   })
+
+  test('should get modal by test id', async() => {
+    render(<Modal {...props} dataTestId="modal-test" />)
```

Comment:
We cannot trust it to go if it is not documented by antd, because it could quietly break compatibility

Reply:
Same for every component that use data-testid directly on antd